x11-packages/gnucash/build.sh

```diff
@@ -0,0 +1,17 @@
+TERMUX_PKG_HOMEPAGE="https://gnucash.org"
+TERMUX_PKG_DESCRIPTION="Personal and small-business financial-accounting software"
+# Licenses: GPL-2.0-or-later, GPL 2 or 3, public domain, special exception for
+# linking OpenSSL.
+TERMUX_PKG_LICENSE="custom"
+TERMUX_PKG_MAINTAINER="@acozzette"
```

Comment:
This may not be obvious from the contribution guide, but you do not have to put yourself down as a maintainer for packages you are submitting.

The Maintainer field is intended to give us a point of contact for build issues or major proposed changes to a package.

We also usually prefer it to be formatted in the Git style `Pseudonym <user@email.tld>` format.
e.g.
```suggestion
TERMUX_PKG_MAINTAINER="@acozzette <acozzette@google.com>"
```
You can also use your Github provided "user email" for this if you prefer.
###### This can be found under `Account > Settings > Email` if you would prefer using it.<br>And should look something like `43716232+TomJo2000@users.noreply.github.com`